WhaleSpotting/Controllers/WhaleSightingController.cs

```diff
@@ -27,4 +31,28 @@ public IActionResult GetById([FromRoute] int Id)
             return NotFound();
         }
     }
+
+    [HttpPatch("{id}/Approve")]
```

Comment:
Can we make this lowercase and just for consistency swap the order?
```suggestion
    [HttpPatch("approve/{id}")]
```

Reply:
changed